src/bottom-sheet.ts

```diff
@@ -220,3 +234,62 @@ export const bottomSheetInfo = async ({
         ],
     });
 };
+
+export const bottomSheetActions = async ({
+    title,
+    subtitle,
+    description,
+    button,
+    secondaryButton,
+    link,
+}: {
+    title?: string;
+    subtitle?: string;
+    description?: string;
+    button: {
+        text: string;
+    };
+    secondaryButton?: {
+        text: string;
+    };
+    link?: {
+        text: string;
+        chevron?: boolean;
```

Comment:
Perhaps we should use `withChevron` instead of `chevron`, in order to align it with the prop's name in ButtonLink